content/api/rest/kom-i-gang/_index.md

```diff
@@ -95,7 +95,7 @@ API nøkkelen må også legges ved i HTTP header slik:
 ApiKey: myKey
 ```
 
-API nøkkel får du etter [registrering av din applikasjon](../../kom-i-gang/#registrer-din-applikasjon).
```

Comment:
Foreslår at du legger inne en lenke, noe ala. 

For mer informasjon, se [Kom i gang med REST-API](../../kom-i-gang/).

Reply:
Det blir jo bare en lenke til toppen av den samme siden, og var usikker på om vi ønsket å sende dem i en loop.

Reply:
Ja, sant. Da er det bedre å lenke til søknadsskjemaet for å søke om API-nøkkel.

Reply:
Vi har ikke noe "registrering av applikasjon" utover at du kan fylle ut et skjema for å søke om API-nøkkel, så teksten burde reflektere dette.

Reply:
Det kan fikses